tracer/src/Datadog.Tracer.Native/interop.cpp

```diff
@@ -79,22 +134,46 @@ EXTERN_C VOID STDAPICALLTYPE InstrumentProbes(
     debugger::DebuggerRemoveProbesDefinition* revertProbes,
     int revertProbesLength)
 {
-    return trace::profiler->InstrumentProbes(methodProbes, methodProbesLength, lineProbes, lineProbesLength, spanProbes, spanProbesLength, revertProbes, revertProbesLength);
+    if (trace::profiler == nullptr)
+    {
+        trace::Logger::Error("Error in InstrumentProbes call. Tracer CLR Profiler was not initialized.");
+        return;
+    }
+
+    trace::profiler->InstrumentProbes(methodProbes, methodProbesLength, lineProbes, lineProbesLength, spanProbes, spanProbesLength, revertProbes, revertProbesLength);
 }
 
 EXTERN_C int STDAPICALLTYPE GetProbesStatuses(WCHAR** probeIds, int probeIdsLength, debugger::DebuggerProbeStatus* probeStatuses)
 {
+    if (trace::profiler == nullptr)
+    {
+        trace::Logger::Error("Error in GetProbesStatuses call. Tracer CLR Profiler was not initialized.");
+        return 0;
```

Comment:
I'm assuming this is the right value to return 😄 🤷‍♂️

Reply:
In `tracer\src\Datadog.Tracer.Native\debugger_probes_instrumentation_requester.cpp`, if `probeIds == nullptr` then we return 0, so LGTM